integration-tests/bats/index.bats

```diff
@@ -2112,13 +2112,6 @@ SQL
     [[ "$output" =~ "pk1" ]] || false
     [[ "$output" =~ "5" ]] || false
     [[ "${#lines[@]}" = "2" ]] || false
-    dolt sql <<SQL
```

Comment:
No since, it is an update it shouldn't error for updating the same value again and again. The -r test below validates than insert can toggle a duplicate key error